services/motion/builtin/builtin.go

```diff
@@ -119,11 +119,8 @@ func (ms *builtIn) Reconfigure(
 		return err
 	}
 	if config.LogFilePath != "" {
-		logger, err := utils.NewFilePathDebugLogger(config.LogFilePath, "motion")
-		if err != nil {
-			return err
-		}
-		ms.logger = logger
+		fileAppender, _ := logging.NewFileAppender(config.LogFilePath)
```

Comment:
This is not an error we are ignoring here; this is the logger itself as an io.Closer, which we want to ignore as closing it would also close `stdout`.